bench/wtperf/wtperf.h

```diff
@@ -409,10 +409,11 @@ static inline char *
 dstrndup(const char *str, const size_t len)
 {
 	char *p;
+	p = dcalloc(len + 1, 1);
```

Comment:
Could you add a comment explaining why we don't call strndup - I can see my self changing this back in the future.